src/controllers/albums.js

```diff
@@ -0,0 +1,112 @@
+// might need to redo these
```

Comment:
If you have things that you may like to look at again in the future, I would recommend putting them in the README with a sentence or bullet points on what you would like to update and why. It keeps things open, and also means you are less likely to not forget about it compared to a single line deep in the repository you wont see unless you come specifically to this file